@xen-orchestra/web-core/lib/locales/en.json

```
@@ -577,7 +577,7 @@
   "vm-status.suspended": "Suspended",
   "vm.active": "Active",
   "vm.inactive": "Inactive",
-  "vms": "VMs",
+  "vms": "VM | VMs",
```

For plural, add 0, 1, and many variants. ` 0 | 1 | many`
```suggestion
  "vms": "VM | VM | VMs",
```